src/main/java/com/labsynch/labseer/service/StandardizationServiceImpl.java

```diff
@@ -212,30 +231,40 @@ public int populateStanardizationDryRunTable()
 					nonMatchingCmpds++;
 				}
 			}
-			cdId = chemStructureService.saveStructure(stndznCompound.getMolStructure(), StructureType.DRY_RUN, false);
+			cdId = saveDryRunStructure(stndznCompound.getMolStructure());
 			if (cdId == -1) {
 				logger.error("Bad molformat. Please fix the molfile for Corp Name " + stndznCompound.getCorpName()
 						+ ", Parent ID " + stndznCompound.getParentId() + ": " + stndznCompound.getMolStructure());
 			} else {
 				stndznCompound.setCdId(cdId);
 				stndznCompound.persist();
 			}
+
+			if (p % 100 == 0){
+				logger.debug("flushing loader session");
+				session.flush();
+				session.clear();
+			}
 			// Compute your percentage.
 			percent = (float) Math.floor(p * 100f / totalCount);
 			if (percent != previousPercent) {
+				currentTime = new Date().getTime();
 				// Output if different from the last time.
-				logger.info("populating standardization dry run table " + percent + "% complete");
+				logger.info("populating standardization dry run table " + percent + "% complete (" + p + " of "
+				+ totalCount + ") average speed (rows/min):"+ (p/((currentTime - startTime) / 60.0 / 1000.0)));
+				currentTime = new Date().getTime();
+				logger.debug("Time Elapsed:"+ (currentTime - startTime));
 			}
 			// Update the percentage.
 			previousPercent = percent;
 			p++;
+
 		}
 		logger.info(
 				"total number of non matching; structure, display or as drawn display changes: " + nonMatchingCmpds);
 		return (nonMatchingCmpds);
 	}
 
-	@Transactional
```

Comment:
Curious about why @Transactional was removed on this function, but maybe was not necessary if the aliases are already populated on the `parent` object.